src/translib/transformer/xfmr_vrf.go

```diff
@@ -334,7 +334,7 @@ var network_instance_table_name_xfmr TableXfmrFunc = func (inParams XfmrParams)
                 tblList = append(tblList, NwInstTblNameMapWithName[intNwInstName])
         }
 
-        log.Info("network_instance_table_name_xfmr, OP ", inParams.oper)
+	xfmrLogInfoAll("network_instance_table_name_xfmr, OP :%v", inParams.oper)
         log.Info("network_instance_table_name_xfmr,  DB table name ", tblList)
```

Comment:
We initially made changes to the logs where parameters were printed from inParams. But doing this only when we print dbDataMap in inParams only as this can contains huge data.